crawl-ref/source/dat/descript/monsters.txt

```diff
@@ -2324,6 +2361,11 @@ steam dragon
 A relatively small grey dragon, with a resilient-looking scaly hide and steam
 pouring from its mouth.
 %%%%
+stoker
+
+A blacksmith who made an unholy pact in life, now chained to tend the fires of
```

Comment:
It seems weird that so many blacksmiths made unholy pacts. Maybe just 'a mortal'?